airbyte-cdk/java/airbyte-cdk/dependencies/src/main/resources/log4j2-test.xml

```diff
@@ -13,53 +13,48 @@
 
     <Appenders>
         <Console name="JvmLogsStdOut" target="SYSTEM_OUT">
-            <ContextMapFilter onMatch="DENY" onMismatch="ACCEPT">
-                <KeyValuePair key="simple" value="true"/>
-            </ContextMapFilter>
             <PatternLayout pattern="${jvm-log-pattern}"/>
         </Console>
-
         <Console name="ContainerLogsStdOut" target="SYSTEM_OUT">
-            <ContextMapFilter onMatch="ACCEPT" onMismatch="DENY">
-                <KeyValuePair key="simple" value="true"/>
-            </ContextMapFilter>
             <PatternLayout pattern="${container-log-pattern}"/>
         </Console>
-
         <File name="JvmLogsFile" fileName="${logDir}/airbyte_jvm.log">
-            <ContextMapFilter onMatch="DENY" onMismatch="ACCEPT">
-                <KeyValuePair key="simple" value="true"/>
-            </ContextMapFilter>
-            <PatternLayout pattern="${jvm-log-pattern}"/>
+            <PatternLayout disableAnsi="true"  pattern="${jvm-log-pattern}"/>
```

Comment:
removing ANSI coloring from non-unified logs. We still have the coloring for unified logs